pkl/private/pkl_codegen_java.bzl

```diff
@@ -122,6 +122,7 @@ def pkl_java_library(name, srcs, module_path = [], generate_getters = None, deps
         module_path: List of Java module targets. Must export provide the JavaInfo provider.
         generate_getters: Generate private final fields and public getter methods instead of public final fields. Defaults to True.
         deps: Other targets to include in the Pkl module path when building this Java library. Must be pkl_* targets.
+        pkl_deps: Targets to include in the Java library as deps. Must be `JavaInfo` targets.
```

Comment:
Should this be called `java_deps` instead?

Reply:
`pkl_deps` suggests to me that these are expected to be Pkl dependencies instead of Java dependencies. I think `java_deps` is a lot more clear here

Reply:
Agreed, I have the same intuition as @bioball regarding the name